source/guide_neos.rst

```diff
@@ -0,0 +1,144 @@
+.. sectionauthor:: Daniel Kratz <uberlab@danielkratz.com>
+.. highlight:: console
+
+.. sidebar:: Logo
+  
+  .. image:: _static/images/neos.png 
+      :align: center
+
+#########
+Neos
+#########
+
+Neos_ is an open source Content Application Platform based on its own PHP framework Flow and distributed under the GPLv3 licence. 
+
+The system is best known for it's intuitive approach of editing content directly in the website (also known as frontend editing) and the mighty marketing features like content dimensions which allow to display optimized contents for different target audiences. 
+
+Neos (formerly TYPO3 Neos) was released for the first time in 2013. It is maintained by the Neos-Team and Contributors.
+
+
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of 
+
+  * PHP_
+  * MySQL_ 
+  * domains_
+
+Prerequisites
+=============
+
+We're using PHP_ in the stable version 7.1: 
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.1'
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+Your website domain needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Since Neos uses the subdirectory Web/ as web root you should not install Neos in your `document root`_. Instead we install it next to that and then use a symlink to make it accessible.
+
+``cd`` to one level above your `document root`_, then use the dependency manager Composer to create a new project based on the Neos base distribution:
+
+.. note:: Composer will install all neccessary dependencies Neos needs to run. This can take some time.
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/
+ [isabell@stardust isabell]$ composer create-project neos/neos-base-distribution Neos
+ Neos
+ Installing neos/neos-base-distribution (42.23.1)
+   - Installing neos/neos-base-distribution (42.23.1): Downloading (100%)
+ Created project in Neos
+ Loading composer repositories with package information
+ Updating dependencies (including require-dev)
+ [...]
+
+ [isabell@stardust isabell]$
+
+Remove your unused `document root`_ and create a new symbolic link to the Neos/Web directory:
+
```

Comment:
Maybe include a warning (..warning:: …) for users who already have contents in html?

Reply:
I added a warning before removing the document root (9daf5e3).